turbo.json

```diff
@@ -27,22 +27,29 @@
     },
     "build:scaffold": {
       "outputs": ["dist/**"],
-      "dependsOn": ["^build:common", "^build:core", "^build:ui", "^build:wallet", "^build:siwe"],
+      "dependsOn": [
+        "^build:common",
+        "^build:core",
+        "^build:ui",
+        "^build:wallet",
+        "@web3modal/siwe#build:siwe",
```

Comment:
@glitch-txs Is this because we only want to build `siwe`, not the dependencies of the `siwe`?

Reply:
This is related to my comment:

> added "@web3modal/siwe#build:siwe", as dependsOn string for scaffold: peerDeps are not considered by turbo (see https://github.com/vercel/turbo/issues/1601#issuecomment-1805589927)

During build turbo wasn't wating for SIWE to be build before starting to build Scaffold which was causing issues, it was because Turbo ignores peerDeps, this fixes it